compiler/lib/js_output.ml

```diff
@@ -456,6 +456,33 @@ struct
     in
     traverse l e
 
+  let stop_on_statement st =
```

Comment:
Maybe add a comment here 
```
The debuggers do not stop on some statements, like function
declarations. So there is no point in outputting some debug information
there.
```

Where does this info come from ?

Reply:
I did some tests.